README.md

```diff
@@ -54,7 +54,7 @@ binary or structured format, and determine what version of the CloudEvents
 specification you want to send the event as.
 
 By default, the `HTTPEmitter` will emit events over HTTP POST using the
-1.0 specification, in binary mode. You can emit 0.3 events by providing
+latest supported specification, in binary mode. You can emit version specific events by providing
```

Comment:
Suggestion: "latest supported specification version"

Reply:
yea, that makes sense